securedrop/dockerfiles/focal/python3/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN apt-get update && apt-get install -y paxctl && \
     apt-get install -y apache2-dev coreutils devscripts vim \
                        python3-pip python3-all python3-venv virtualenv libpython3.8-dev libssl-dev \
                        gnupg2 ruby redis-server git xvfb curl wget \
-                       gettext paxctl x11vnc enchant libffi-dev sqlite3 gettext sudo \
+                       x11vnc enchant libffi-dev sqlite3 gettext sudo \
```

Comment:
Am I totally missing it or is paxctl not duplicated in this one and now missing?

Reply:
`paxctl` was only present in the dockerfile to support folks who were running grsecurity-patched kernels on their development workstations. These days I believe that number is zero, so it's OK in my view to remove paxctl from the container images entirely.

Reply:
Got it, perfect then!

Reply:
For this file, currently it's still in the first line of the apt-get commands, by itself.